pydm/exception.py

```diff
@@ -0,0 +1,82 @@
+import io
+import sys
+import traceback
+from collections import namedtuple
+from qtpy import QtWidgets
+
+
+class QtExceptionHandler:
```

Comment:
A bit weird to have a class in Python with everything `@staticmethod` - feels like C++ code 🤷‍♂

Reply:
Addressed via https://github.com/slaclab/pydm/pull/599/commits/1055892aab1ea1661ba4f8dda43ea77ef7e5bf7c